doc/yamlConfigFile.md

```diff
@@ -28,6 +28,13 @@ log:
     mode: roll
 ```
 
+## YAML configuration schema validation
+
+Users can validate YAML configurations file against JSON schema.
+First you should convert YAML file into JSON and then you can
```

Comment:
It is not needed if you use https://marketplace.visualstudio.com/items?itemName=redhat.vscode-yaml . I would suggest documenting such use-case which is more friendly to users